lib/dependencies/HarmonyExportDependencyParserPlugin.js

```diff
@@ -75,10 +79,19 @@ module.exports = class HarmonyExportDependencyParserPlugin {
 				clearDep.loc = /** @type {DependencyLocation} */ (statement.loc);
 				clearDep.loc.index = -1;
 				parser.state.module.addPresentationalDependency(clearDep);
+				const { defer } = getImportMode(parser, statement, this.deferImport);
+				if (defer && statement.type !== "ExportAllDeclaration") {
+					const error = new WebpackError(
+						"Deferred import can only be used with `export * as namespace from '...'` syntax."
```

Comment:
Actually, the proposal now only supports `import defer *` and not `export defer *`.

Reply:
I'll keep it for now since I haven't updated the test suite.